### PR TITLE
store jsons in directory

### PR DESCRIPTION
**Pull request by OlegLoewen**

### CODE REVIEW

File: cli/elastic.py [hunk 1]
```diff
@@ -45,3 +47,10 @@ def store_files(index: str, files: [str], cfg_name: str):
                 body=json.load(fh),
                 )
             )
+
```
Comment (ccwienk) [inline cli/elastic.py]: nit: see pycodestyle hints (I just triggered the PR validation job by adding the ok-to-test-label)
-> two blank lines between functions

File: cli/elastic.py [hunk 2]
```diff
@@ -45,3 +47,10 @@ def store_files(index: str, files: [str], cfg_name: str):
...
+def store_dir(index: str, directory: str, cfg_name: str):
```
Comment (ccwienk) [inline cli/elastic.py]: use util.CliHints.existing_dir() as type (then the existence will be checked automatically)

File: cli/elastic.py [hunk 3]
```diff
@@ -45,3 +47,10 @@ def store_files(index: str, files: [str], cfg_name: str):
...
+    elastic_cfg = util.ctx().cfg_factory().elasticsearch(cfg_name)
+    elastic_client = ccc.elasticsearch.from_cfg(elasticsearch_cfg=elastic_cfg)
+
+    jsonFiles = [join(directory, f) for f in listdir(directory) if (isfile(join(directory, f)) and f.endswith('.json'))]
```
Comment (ccwienk) [inline cli/elastic.py]: consider using os.walk (thus recursing into sub-directories)

**Comment by OlegLoewen** [comment]
Please review again.

**Comment by ccwienk** [comment]
@OlegLoewen : please resolve conflicts..

**Pull request closed without merging**